otter/cloud_client.py

```diff
@@ -507,6 +524,25 @@ class NovaComputeFaultError(Exception):
     """
 
 
+@attributes([])
```

Comment:
Hmm, you're right, it shouldn't be needed.  Not sure why I put that there - will delete.  Thanks!  :)